webapp/app/controllers/feed_controller.rb

```diff
@@ -51,6 +51,7 @@ def posts
 		    		{
 					    author_name: post.author_name,
 					    link: post.link,
+							author_link: (post.author==nil) ? nil : user_path(post.author),
```

Comment:
Consigli di houndci sono buoni!
C'è poi un altro problema, tipico di rails, di cui di solito ci si rende conto dopo aver letto la guida di rails un paio di volte :P C'è una riga di codice più in alto da cambiare per risolvere l' "n+1 queries problem", e migliorare di molto le performance, leggi qui per una spiegazione del problema e come risolverlo http://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations